charts/actions-runner-controller/crds/actions.summerwind.dev_runnerdeployments.yaml

```diff
@@ -32,6 +32,10 @@ spec:
         - jsonPath: .metadata.creationTimestamp
           name: Age
           type: date
+        - jsonPath: .spec.template.spec.labels
```

Comment:
This needs to be generated via [crd-gen markers](https://github.com/actions/actions-runner-controller/blob/7d0918b6d5ac8f64d44caa9ea133f7089dc2b243/apis/actions.summerwind.net/v1alpha1/runnerdeployment_types.go#L80-L84) and `make manifests`!
Would you mind giving it a shot?